azavea/temperate
# Allow configuration of WeatherEvent to determine user-facing theming

## Overview
Adds a multiple-choice option to WeatherEvent objects to determine what CSS class they should use, which helps determine what image they should be displayed with along with potentially other factors such as text color and positioning.

### Demo
<img width="1265" alt="screen shot 2017-12-12 at 4 48 22 pm" src="https://user-images.githubusercontent.com/1032849/33910320-5b210b9e-df5c-11e7-8fb9-88bd2d535baf.png">

## Notes
- WeatherEvents without concerns are not currently styled, making their label difficult to read.

## Testing Instructions
- Install migrations
- Use `./scripts/manage loaddata weatherevents` to import weather event configuration
- Ensure your organization has WeatherEvents with http://localhost:8100/api/weather-event-rank/
- In http://localhost:4210/indicators, you should see a list of Concerns with varying background images

Closes #252
